src/lib/POWERMGNT/POWERMGNT.h

```diff
@@ -51,13 +51,13 @@
 #endif
 
 #ifdef TARGET_TX_ESP32_E28_SX1280_V1
-#define MaxPower 4
-#define DefaultPowerEnum 2
+#define MaxPower PWR_500mW
```

Comment:
E28 module cannot push out full 500mW so it is limited to 250mW. Please fix.

Reply:
@cruwaller, doesn't E28-2G4M27S module push out 500mW? I looked for in Ebyte website ([https://www.ebyte.com/en/product-class.html?key=e28](https://www.ebyte.com/en/product-class.html?key=e28))
and this is what I found:
<img width="910" alt="Schermata 2021-03-25 alle 12 38 03" src="https://user-images.githubusercontent.com/9280405/112467290-2bbebb00-8d67-11eb-9d54-da34865ba9f4.png">

Reply:
All current power levels have been confirmed.  The e28, gnice, and even the RF amp used in the r9 don't reach their published spec.

Reply:
Changed to 250mW